xml/virt_support.xml

```diff
@@ -1030,8 +1030,27 @@
   </sect1>
   <sect1 xml:id="virt-support-feature-support">
     <title>Feature support</title>
-
     <para/>
+    <title>Nested virtualization: Tech Preview</title>
+    <para>
+      Nested virtualization allows you to run a virtual machine inside another
+      VM while still using hardware acceleration from the host. It has low
+      performance and adds more complexity while debugging. Nested
+      virtualization is normally used for testing purposes. In &sls;, nested
+      virtualization is a Technology Preview. It is only provided for testing
+      purposes and is not supported. Bugs can be reported but they are treated
+      with low priority. Any attempt to live-migrate or to save or restore VMs
+      in the presence of nested virtualization is also explicitly unsupported.
+    </para>
+    <title>Post-Copy Live migration: Tech Preview</title>
+    <para>
+     Post-Copy is a method to live migrate Virtual Machines that is intended to get VMs running as soon as possible on the destination host,
+     and have the VM RAM be transferred gradually in the background over time as needed.
+     Under some conditions, this can be an optimization compared to the traditional Pre-Copy method.
+     However this comes with a major drawback: an error occurring during the migration (especially a network failure) can cause the whole VM RAM contents to be lost.
+     
+     Therefore, it is recommended to only use Pre-Copy in production, while Post-Copy can be used for testing and experimentation in case losing the VM state is not a major concern.
```

Comment:
```suggestion
     Therefore, we recommend using pre-copy only in production, while post-copy can be used for testing and experimentation in case losing the VM state is not a major concern.
```